ide/css.lib/src/org/netbeans/modules/css/lib/Css3.g

```diff
@@ -2046,5 +2046,5 @@ LINE_COMMENT
     ;
 
 // -------------
-//  Illegal.    Any other character shoudl not be allowed.
+//  Illegal.    Any other character should not be allowed.
```

Comment:
You're changing a grammar file. Did you regenerate the grammar afterwards?

Reply:
This is comment line

Reply:
Do not change the grammar if there is no error to fix. I question fixing typos in non-user-facing code in general and in this case I plain object it.